wordpress/wp-content/plugins/cds-web-blocks/resources/js/src/product/components/sidebar.js

```diff
@@ -0,0 +1,25 @@
+const { PanelBody } = wp.components;
+const { __ } = wp.i18n;
+import ToggleControl from "../../components/ToggleControl";
+import SelectControl from "../../components/SelectControl";
+
+export const ProductFields = () => {
+    return (
+        <div>
+            <PanelBody title={__("Status", "cds-web")} initialOpen={true}>
+                <ToggleControl label={__("On Homepage", "cds-web")} metaKey="cds_product:on_home_page" />
+                <SelectControl
+                    options={[
+                        { label: 'Discovery', value: 'Discovery' },
+                        { label: 'Alpha', value: 'alpha' },
+                        { label: 'Beta', value: 'beta' },
+                        { label: 'Live', value: 'live' },
+                    ]}
+                    label={__("Phase", "cds-web")}
+                    metaKey="cds_product:phase"
+                />
```

Comment:
Not sure if there's anything to be done here about defaults (or adding blank values so that no default is expected (which would only work for the select and text field)), what do you think?

Reply:
Added a select one option - moving forward will need to figure out how to save a default value without the onChange